src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -106,6 +107,16 @@ def get_limit():
             return ()
 
     def ansible_playbook(environment, playbook, *cmd_args):
+        # verifying cchq env ansible version before execution
+        required_ansible_version = environment.public_vars['ansible_version']
```

Comment:
Can this just be hard-coded? Taking a step back, the only reason we are going the extra mile and adding in this warning is that due to what you might call a mistake by the ansible maintainers, the normal steps for updating ansible from version X to version Y don't work, and you have to do this additional uninstall step first.

If you just hard code `required_ansible_version` to what today (Aug 5) is the version you're trying to upgrade to that is fine, because that's the version before which this upgrade process requires this manual step.